datashuttle/utils/folders.py

```diff
@@ -437,16 +446,22 @@ def search_for_folders(  # TODO: change name
     local_or_central : "local" or "central"
     search_path : full filepath to search in
     search_prefix : file / folder name to search (e.g. "sub-*")
+    verbose : If `True`, if a search folder cannot be found, a message
```

Comment:
Same here:
```suggestion
    verbose : If `True`, when a search folder cannot be found, a message
          will be printed with the missing path.
```

Reply:
Same for other occurrences of this docstring as well (if you prefer my wording that is)